### PR TITLE
feat: create PR template

### LINKED ISSUE #22: PR template
Please create a PR template for this repository so that people can submit better PRs.

### PR DESCRIPTION
**Pull request by Jonniie**
# Pull Request Template

## Fixes Issue

**Closes #22**

## Proposed Changes
Added a pull request template for consistent and organized contributions. 

## Checklist

Please check all the applicable boxes. To mark a box as done, use the following conventions:
```
- [x] - Correct; marked as done
- [ ] - Not correct; marked as **not** done
```
- [ ] My code follows the code style of this project.
- [x] I've written clear and short commit messages.
- [x] The title and description of the PR are clear and explain the approach.
- [x] I have performed a self-review of the code.
- [x] I have added proper comments wherever needed.
- [x] I have run my code through prettier to ensure consistency.

## Notes to Reviewers

Please provide any additional context or information for the reviewers. If there are specific areas or concerns you'd like the reviewers to focus on, mention them here.

## Screenshots

If your changes include any visual or UI updates, add relevant screenshots to help reviewers understand the changes better.

## Test Plan

If applicable, describe the steps you've taken to test your changes.

## Related Issues

List any related issues or pull requests that are addressed or linked to this PR.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- [ ] My code follows the code style of this project.
+- [ ] My commit messages are clear and concise
+- [ ] The title and description of the PR are clear and explain the approach.
+- [ ] I have performed a self-review of the code.
+- [ ] I have added proper comments wherever needed.
+- [ ] I have run my code through prettier to ensure consistency.
+
+## Notes to Reviewers
+
+Please provide any additional context or information for the reviewers. If there are specific areas or concerns you'd like the reviewers to focus on, mention them here.
```
Comment (manancodes) [inline .github/pull_request_template.md]: Can we add these as comments, just to make sure this does not get added in the PR itself. Same for the rest of the helper texts.

**Comment by manancodes** [comment]
@Jonnie-Dev

**Comment by Jonniie** [comment]
What reviews did you suggest, I'm finding it hard to find them here

**Comment by manancodes** [comment]
Screenshot 2023-10-22 at 4 28 26 PM

**Comment by Jonniie** [comment]
Oooh, my initial thought was for the contributor to actually edit the texts

**Comment by manancodes** [comment]
@Jonnie-Dev Any update on this?

**Comment by Jonniie** [comment]
yeah, ive commented out the help notes

**Review by manancodes — APPROVED** [review]

**Comment by manancodes** [comment]
LGTM

**Pull request merged**